src/helper/compression.cpp

```diff
@@ -11,17 +11,18 @@
 
 namespace dds {
 
-static constexpr int64_t encoding = -0xf;
-static constexpr int max_round_decompression = 100;
-
+namespace {
+constexpr int64_t encoding = -0xf;
+constexpr int max_round_decompression = 100;
 // Taken from PHP approach
 // https://heap.space/xref/PHP-7.3/ext/zlib/php_zlib.h?r=8d3f8ca1#36
-constexpr size_t COMPRESSED_SIZE_ESTIMATION(size_t in_len)
+static size_t estimate_compressed_size(size_t in_len)
```

Comment:
```suggestion
size_t estimate_compressed_size(size_t in_len)
```
The anonymous namespace is equivalent to static in this case.